src/Logging.hs

```diff
@@ -51,18 +51,20 @@ module Logging
   , AppName(..)
   , unAppName
   , showAppName
+  , parseAppName
   -- * Logger
   , AppNameLogger
   ) where
 
 import           Control.Lens
 import qualified Control.Monad.Log             as L
-import qualified Data.String          -- required for IsString instance.
+import qualified Data.String                   as Str  -- required for IsString instance.
 import qualified Data.Text                     as T
-import "protolude" Protolude                  
+import qualified Options.Applicative           as A
+import "protolude" Protolude
 
 -- | A type alias for convenience
```

Comment:
What do you think about adding something like "A Control.Monad.Log Logger where the environment is an AppName." ?